mod-invoice-storage/schemas/invoice.json

```diff
@@ -0,0 +1,147 @@
+    },
+    "approvedBy": {
+      "description": "UUID of user that approved this invoice",
+      "type": "string",
+      "pattern": "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$"
+    },
+    "approvalDate": {
+      "description": "Date the invoice was approved for processing",
+      "type": "string",
+      "format": "date-time"
+    },
+    "chkSubscriptionOverlap": {
+      "description": "IF TRUE the system will check if there is another invoice for this subscription and whether the dates overlap. IF the dates overlap, the system should issue an alert.",
+      "type": "boolean"
+    },
+    "currency":{
+      "description": "Ideally this is the ISO code and not something the user defines",
+      "type": "string"
+    },
+    "folioInvoiceNo": {
+      "description": "Invoice number in folio system",
+      "type": "string"
+    },
+    "invoiceDate": {
+      "description": "Invoice date",
+      "type": "string",
+      "format": "date-time"
+    },
+    "note": {
+      "description": "Invoice note",
+      "type": "string"
+    },
+    "owner": {
+      "description": "The location/branch/division etc that's responsible for this invoice. This may or may not be the same as the PO. ie. North Branch",
+      "type": "string",
+      "pattern": "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$"
+    },
+    "paymentDue": {
+      "description": "When this is required to be paid. Generally governed by the relationship with the Vendor",
+      "type": "string",
+      "format": "date-time"
+    },
+    "paymentTerms": {
+      "description": "Invoice payment terms",
+      "type": "string"
+    },
+    "paymentMethod": {
+      "description": "Inherited from vendor record",
+      "type": "string"
+    },
+    "prorata": {
+      "description": "IF TRUE the additional charges will be applied to each line item and reflected in the total. IF FALSE each charge will be added to the invoice as it's own line item. (Default would be Yes) Ideally this can be done by price.",
+      "type": "boolean",
+      "default": true
+    },
+    "status": {
+      "description": "Open: Record has been created, Reviewed: details have been verified, Approved: Funds are release, Paid: confirmation that funds have been exchanged and check number has been returned amounts are frozen, cancelled.\nNote: invoices are never partially paid.",
+      "type": "string",
+      "enum": [
+        "Open",
+        "Reviewed",
+        "Approved",
+        "Paid",
+        "Cancelled"
+      ]
+    },
+    "source": {
+      "description": "This does not denote a user ID but describes how the record was created. Eg. Manually, EDIFACT, MARC etc",
+      "type": "string",
+      "pattern": "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$"
+    },
+    "totalAmount": {
+      "description": "Total amount",
+      "type": "integer"
```

Comment:
I believe this should be defined as `number` like in the [invoice.schema](https://github.com/folio-org/deprecated-mod-acquisitions/blob/master/ramls/_schemas/invoice.schema)